src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -256,10 +256,10 @@
     <value>DAC properties cannot contain method invocations</value>
   </data>
   <data name="PX1030FixBound" xml:space="preserve">
-    <value>For a bound field, set PersistingCheck of PXDefaultAttribute to Nothing</value>
+    <value>Set PersistingCheck of PXDefaultAttribute to Nothing</value>
   </data>
   <data name="PX1030Title" xml:space="preserve">
-    <value>The PXDefault attribute of the field is used incorrectly</value>
+    <value>The PXDefault attribute of the existing field is used incorrectly. PXDefaultAttribute used on a custom field defined in PXCacheExtension can potentially prevent updates to existing records when used without PersistingCheck property set to Nothing</value>
```

Comment:
@k-s-popova , please verify this string

Reply:
@dropsonic @SENya1990 , verify this string

Reply:
I'll remove 'existing'

Reply:
"existing" should be removed